readme.md

```diff
@@ -126,6 +126,8 @@ When a namespace with the opt-in annotation is created, the following happens:
 4. the relative Azure machine is assigned the additional IP on the primary NIC (support for secondary interfaces in not available).
 5. the relative `hostsubnet` is updated to reflect the assigned IP, the `egressIP` field is updated.
 
+By default, the Egress IP address is added to the members of the backend pool of the load balancer. The determination of whether to attach this assignment is configured through the `egressip-ipam-operator.redhat-cop.io/azure-load-balancer-attach=<true|false>` annotation.
```

Comment:
Not a lot of skin in the game here, but the docs here do not mention why it would do this in the first place on Azure as using EgressIPs has no bearing on the Ingress VIP (load balancer) in this case. Are there any Azure docs that could be linked here to have more information around this for future users of this operator?